config/database.yml

```diff
@@ -7,3 +7,11 @@ development:
   database: task_manager_development
   username: postgres
   password: postgres
+
+test:
+  <<: *default
+  adapter: postgresql
+  database: task_manager_development
```

Comment:
why you are using same database for development and test env ?

Reply:
done